Sources/Swift/Protocol/SentryLogLevel.swift

```diff
@@ -0,0 +1,43 @@
+@objc
+public enum SentryLogLevel: Int {
+    case trace
+    case debug
+    case info
+    case warn
+    case error
+    case fatal
+    
+    public var value: String {
+        switch self {
+        case .trace:
+            return "trace"
+        case .debug:
+            return "debug"
+        case .info:
+            return "info"
+        case .warn:
+            return "warn"
+        case .error:
+            return "error"
+        case .fatal:
+            return "fatal"
+        }
+    }
+    
+    public func toSeverityNumber() -> Int {
```

Comment:
`l`: Please add a reference to the docs so we can find the source of the numbers again in the future.